index.html

```diff
@@ -0,0 +1,205 @@
+</head>
+<body>
+   <div class="nav"> 
+    <header> 
+        <a class=" logo" href="#"><h1>Welcome</h1></a> 
+    </header>
+    <nav>
+        <a href="#"><img id="MenuIcons" src="images/Icons/Menu.png" alt="togglebutton"></a>
+    </nav>
+    </div>
+    <main >
+        <section id="section-1">
+                <div class="Primary"><h2 class="Primary-1">Hey there. I’m Mariane</h2>
+                        <h2 id="Primary-2" >I’m a software developer</h2></div>
+                        <div class="supporting"> <p>I can help you build a product , feature or website Look through some of my work 
+                        and experience! If you like what you see and have a project you need coded,<br> 
+                        don’t hestiate to contact me.</p>
+                </div>
+                
+                <ul class=social>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/twitter.png" alt="Twitter icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/Linkedinicon.png" alt="Linkedin icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/medium.png" alt="medium icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/github.png" alt="github icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/angellist.png" alt="angellist icon"></a></div></li>
+                </ul>
+                
+               <div class="action"><a href="#">
+               <img src="images/Icons/Disabled.png" alt="Disabled"></a></div>
+        </section>
+
+            <div class="Primary"><h2 class="Primary-1">My Recent Works</h2><hr></div>
+        <section id="section-2" class="grid">
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
```

Comment:
Same thing here
- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 

```
 <ul class="cells">
      <li> <a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a>></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
</ul>
```